src/index.html

```diff
@@ -12,20 +12,43 @@
     <link
         href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
         rel="stylesheet">
-    <script src='main.js'></script>
 </head>
 
 <body>
-    <div class="overlay">
-
+    <div id="sidepan" class="sidepanel">
+        <a id="closebtn" href="javascript:void(0)" class="close-btn">&times;</a>
```

Comment:
If you assign a class 'ban' to this tag, MAYBE YOU ALSO NEED TO USE `button` AS WELL???)